src/proto/FunctionRpc.proto

```diff
@@ -337,6 +337,9 @@ message RpcFunctionMetadata {
   // A flag indicating if managed dependency is enabled or not
   bool managed_dependency_enabled = 14;
 
+  // Current retry context
+  RetryContext retry_context = 15;
```

Comment:
This is to replace what is currently specified in a function.json file, right? If so, I think we need a new type and different naming because "retry context" is for info specific to the invocation. Based on [these docs](https://learn.microsoft.com/azure/azure-functions/functions-bindings-error-pages?tabs=fixed-delay%2Cin-process&pivots=programming-language-python#retry-examples) they seem to call it the "retry policy" and in function.json it was the "retry" property, so I would do this:

```suggestion
  RetryPolicy retry = 15;
```

Obviously then we'd need to define the `RetryPolicy` type in this file as well

Reply:
Retry context is not specific to the invocation but for the entire function. This is going to be the same retry context that the worker receives in the invocation request. 
![image](https://user-images.githubusercontent.com/80794152/221051369-2396c082-6ce1-4e75-899d-82737c630ca3.png)
 Just that in the v2 model since the worker indexes the function, we need a way to send this info to the host.

Reply:
Fixed it.